server/sonar-server/src/test/java/org/sonar/server/issue/ws/SearchActionMediumTest.java

```diff
@@ -609,14 +615,24 @@ public void display_deprecated_debt_fields() throws Exception {
     tester.get(IssueIndexer.class).indexAll();
 
     userSessionRule.login("john");
-    WsTester.Result result = wsTester.newGetRequest(IssuesWs.API_ENDPOINT, SearchAction.SEARCH_ACTION)
+    WsTester.Result result = wsTester.newGetRequest(API_ENDPOINT, SEARCH_ACTION)
       .setParam("resolved", "false")
       .setParam(WebService.Param.FACETS, "severities")
       .setParam("facetMode", DEPRECATED_FACET_MODE_DEBT)
       .execute();
     result.assertJson(this.getClass(), "display_deprecated_debt_fields.json");
   }
 
+  @Test
+  public void fail_when_invalid_format() throws Exception {
```

Comment:
Next time, please do not add tests in class using ServerTester but create another test class that's using EsTester and DbTester